### DISCUSSION TITLE
期待OIDC的配置模板和教程

### DISCUSSION THREAD

**Discussion by tatuke** [hhyo/Archery#2070] [General]
期待OIDC的配置模板和教程

**Comment by zhangzhaox** [comment]
已经部署了V1.10.0版本，也不知道怎么开启OIDC功能，麻烦出个配置手册啥的，谢谢

**Reply by arthas3014** [reply]
你好我想问下配置OIDC回调的路由url应该是什么啊

**Reply by hkywgz001** [reply]
http://ip:port(有域名换成域名)/oidc/callback
